src/mapml.js

```diff
@@ -1935,7 +2213,9 @@ M.TemplatedImageLayer =  L.Layer.extend({
             axis = inputs[i].getAttribute("axis"), 
             name = inputs[i].getAttribute("name"), 
             position = inputs[i].getAttribute("position"),
-            select = (inputs[i].tagName.toLowerCase() === "select");
+            select = (inputs[i].tagName.toLowerCase() === "select"),
+            min = inputs[i].getAttribute("min") || min,
```

Comment:
Can select have min/max attributes actually?

Reply:
https://html.spec.whatwg.org/multipage/form-elements.html#the-select-element:
> [Global attributes](https://html.spec.whatwg.org/multipage/dom.html#global-attributes)
> `autocomplete` — Hint for form autofill feature
> `disabled` — Whether the form control is disabled
> `form` — Associates the element with a form element
> `multiple` — Whether to allow multiple values
> `name` — Name of the element to use for form submission and in the form.elements API
> `required` — Whether the control is required for form submission
> `size` — Size of the control

Reply:
I guess we might say that because we're working beyond the pale :-), can `select` have `min/max` attributes even in the MapML world, and I think the answer is still "no", as far as I can see a need at least.

Reply:
The loop goes through all the inputs in an extent including `input type="location"`, this was an artifact from the first attempt to derive the bounds from the given inputs. It will be removed in the next commit as it isn't used anymore.